addons/web_editor/views/snippets.xml

```diff
@@ -518,17 +518,17 @@
                         <div class="o_scroll_shapes_basic">
                             <we-title class="">Geometrics</we-title>
                             <we-select-page string="Geometrics">
-                                <we-button data-set-img-shape="web_editor/geometric/geo_square" data-select-label="Square"/>
-                                <we-button data-set-img-shape="web_editor/geometric/geo_shuriken" data-select-label="Shuriken"/>
-                                <we-button data-set-img-shape="web_editor/geometric/geo_diamond" data-select-label="Diamond"/>
+                                <we-button data-set-img-shape="web_editor/geometric/geo_square" data-select-label="Square" data-no-transform="true"/>
+                                <we-button data-set-img-shape="web_editor/geometric/geo_shuriken" data-select-label="Shuriken" data-no-transform="true"/>
+                                <we-button data-set-img-shape="web_editor/geometric/geo_diamond" data-select-label="Diamond" data-no-transform="true"/>
                                 <we-button data-set-img-shape="web_editor/geometric/geo_triangle" data-select-label="Triangle"/>
```

Comment:
> This new option is not available for certain shapes, such as animated shapes and "Device" shapes.

Maybe reformulate this in the commit message, I got confused while testing: it is available on some animated shapes. Also while testing, it is is sometimes not clear to me why the option is not available. What are the reasons? Maybe this should be put in the commit message.

See here, I am not sure why this one has option and not the next one